lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -79,7 +80,7 @@ func (h *VCHCreate) Handle(params operations.PostTargetTargetVchParams, principa
 
 	validator, err := validateTarget(params.HTTPRequest.Context(), d)
 	if err != nil {
-		return operations.NewPostTargetTargetVchDefault(400).WithPayload(&models.Error{Message: err.Error()})
+		return operations.NewPostTargetTargetVchDefault(http.StatusBadRequest).WithPayload(&models.Error{Message: err.Error()})
```

Comment:
I think this is a really good change. Can you file an issue so that we remember to clean this up in the rest of the handlers?

Reply:
i can do it in this pr

Reply:
fixes done